### PR TITLE
Translate "Garbage Collection" page to Bahasa Indonesia

### PR DESCRIPTION
**Pull request by antoooks**
This addresses to https://github.com/kubernetes/website/issues/13929

### CODE REVIEW

**Comment by antoooks** [comment]
Hmm the codenew shortcode template seems to be broken, that explains why netlify deploy test are failing.

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+Peran dari _garbage collector_ Kubernetes adalah untuk menghapus objek tertentu yang sebelumnya mempunyai pemilik, tetapi tidak lagi mempunyai pemilik.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Pemilik dan _dependent_
+
+Beberapa objek Kubernetes adalah pemilik dari objek lainnya. Sebagai contoh, sebuah ReplicaSet adalah pemilik dari sekumpulan Pod. Objek-objek yang dimiliki disebut _dependent_ dari objek pemilik. Setiap objek _dependent_ memiliki sebuah kolom `metadata.ownerReferences` yang menunjuk ke objek pemilik.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/controllers/garbage-collection.md]: nit: change _dependent_ -> dependen
change _independet_ -> independen
menunjuk -> merujuk

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    controller: true
+    blockOwnerDeletion: true
+    kind: ReplicaSet
+    name: my-repset
+    uid: d9607e19-f88f-11e6-a518-42010a800195
+  ...
+```
+{{< note >}}
+Referensi pemilik lintas _namespace_ tidak diperbolehkan oleh desain. Artinya:
+1) _Namespace-scoped dependent_ hanya bisa menspesifikasikan pemilik jika berada di _namespace_ yang sama, dan pemilik bersifat _cluster-scoped_.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/controllers/garbage-collection.md]: _Namespace-scoped dependent_ -> Dependen dengan cakupan namespace
pemilik bersifat _cluster-scoped_ -> pemilik memiliki cakupan kluster

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    blockOwnerDeletion: true
+    kind: ReplicaSet
+    name: my-repset
+    uid: d9607e19-f88f-11e6-a518-42010a800195
+  ...
+```
+{{< note >}}
+Referensi pemilik lintas _namespace_ tidak diperbolehkan oleh desain. Artinya:
+1) _Namespace-scoped dependent_ hanya bisa menspesifikasikan pemilik jika berada di _namespace_ yang sama, dan pemilik bersifat _cluster-scoped_. 
+2) _Cluster-scoped dependent_ hanya bisa menspesifikasikan pemilik bersifat _cluster-scoped_, tetapi tidak berlaku untuk pemilik bersifat _namespace-scoped_.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/controllers/garbage-collection.md]: _Cluster-scoped dependent_ -> Dependen dengan cakupan kluster
pemilik bersifat _namespace-scoped_ -> pemilik memiliki cakupan namespace

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+### Foreground cascading deletion
+
+Pada *foreground cascading deletion*, pertama objek utama akan memasuki keadaan "_deletion in progress_". Pada saat keadaan "_deletion in progress_", kondisi-kondisi berikut bernilai benar:
+
+ * Objek masih terlihat via REST API
+ * `deletionTimestamp` objek telah ditentukan
+ * `metadata.finalizers` objek memiliki nilai `foregroundDeletion`.
+
+ Ketika dalam keadaan "_deletion in progress_", _garbage collector_ menghapus _dependent_ dari objek. Ketika _garbage collector_ telah menghapus semua "_blocking_" _dependent_ (objek dengan `ownerReference.blockOwnerDeleteion=true`), _garbage collector_ menghapus objek pemilik.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/controllers/garbage-collection.md]: nit: please change all _dependent_ to dependen

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+ ### Background cascading deletion
+ 
+ Pada *background cascading deletion*, Kubernetes segera menghapus objek pemilik dan _garbage collector_ kemudian menghapus _dependent_ pada _background_.
+
+ ### Mengatur kebijakan _cascading deletion_
+
+ Untuk mengatur kebijakan _cascading deletion_, tambahkan kolom `propagationPolicy` pada argumen `deleteOptions` ketika menghapus sebuah Object. Nilai yang dapat digunakan adalah "Orphan", "Foreground", atau "Background".
+
+ Sebelum Kubernetes 1.9, kebijakan default _garbage collection_ untuk banyak _resource controller_ adalah **orphan**. Ini meliputi ReplicationController, ReplicaSet, StatefulSet, DaemonSet, dan Deployment. Untuk jenis pada kelompok versi `extensions/v1beta1`, `apps/v1beta1`, dan `apps/v1beta2`, kecuali kamu menspesifikasikan dengan cara lain, objek _dependent_ adalah _orphan_ secara default. Pada Kubernetes 1.9, untuk semua jenis pada kelompok versi `apps/v1`, objek _dependent_ dihapus secara default.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/controllers/garbage-collection.md]: make default italicized

Comment (antoooks) [reply content/id/docs/concepts/workloads/controllers/garbage-collection.md]: fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md (hunk 4 above)
Comment (antoooks) [reply content/id/docs/concepts/workloads/controllers/garbage-collection.md]: fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md (hunk 3 above)
Comment (antoooks) [reply content/id/docs/concepts/workloads/controllers/garbage-collection.md]: fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md (hunk 2 above)
Comment (antoooks) [reply content/id/docs/concepts/workloads/controllers/garbage-collection.md]: fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305

File: content/id/docs/concepts/workloads/controllers/garbage-collection.md (hunk 1 above)
Comment (antoooks) [reply content/id/docs/concepts/workloads/controllers/garbage-collection.md]: fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305

**Comment by irvifa** [comment]
Test seems to be failed in here: https://github.com/kubernetes/website/runs/135681592

**Comment by antoooks** [comment]
> Test seems to be failed in here: https://github.com/kubernetes/website/runs/135681592
> Can you please help me check this?

@irvifa the page failed to find referred file in code snippet, it was fixed on this commit https://github.com/kubernetes/website/pull/14513/commits/4026b94126e317ad14a8c1eb5ef4e00a9c828a18 by adding the referred file to the example directory.

**Comment by girikuncoro** [comment]
@antoooks  Thanks for contributing to Kubernetes! 🎉 

/lgtm
/approve

**Pull request merged**